nuclear/install.py

```diff
@@ -31,6 +36,13 @@ def handle(args: argparse.Namespace) -> int:
     # error handling for when tar_addr is None has already been done
     if tar_addr is not None:
         # download
-        pass
-
+        r = requests.get(tar_addr)
+        # get filename from content-dispoisiton
+        filename = get_dir.get_filename(r.headers.get('content-disposition'))
+        try:  
+            makedirs(get_dir.get_dir_name(tar_addr))
+            open(get_dir.get_dir_name(tar_addr)+"/"+filename,'xb').write(r.content)
```

Comment:
maybe do it using `with` so that the opened file get closed at the end

Reply:
Okay I'll do this